_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -112,7 +113,35 @@ class HealthCareApp extends React.Component {
           },
           childInformation: {},
           annualIncome: {},
-          deductibleExpenses: {}
+          deductibleExpenses: {},
+        },
+
+        // TODO: insuranceInformation should be an array where each row
+        //       contains all the following fields:
+        insuranceInformation: {
+          insuranceInfo: {
+            isCoveredByHealthInsurance: false,
+            insuranceName: undefined,
```

Comment:
yeah, I noticed the issue just yesterday.  With undefined, the attribute is technically not passed, which *I think* means the component is "not controlled" by react (https://facebook.github.io/react/docs/forms.html).

Hmm...maybe `null` is the right answer.  It's both an invalid value and NOT `undefined`.